nf_core/modules/lint/main_nf.py

```diff
@@ -257,9 +257,26 @@ def check_process_section(self, lines, fix_version, progress_bar):
         self.warned.append(("process_standard_label", "Process label unspecified", self.main_nf))
     for i, l in enumerate(lines):
         url = None
-        if _container_type(l) == "bioconda":
```

Comment:
There's another one which has to be changed to "conda" in line 401 (sorry I can't comment there)